charts/policy-controller/README.md

```diff
@@ -68,7 +68,7 @@ The Helm chart for Policy Controller
 | webhook.volumes | list | `[]` |  |
 | leasescleanup.image.pullPolicy | string | `"IfNotPresent"` |  |
 | leasescleanup.image.repository | string | `"cgr.dev/chainguard/kubectl"` |  |
-| leasescleanup.image.version | string | `"latest"` |  |
+| leasescleanup.image.version | string | `"latest-dev"` |  |
```

Comment:
in the description column, can you comment the use of the `-dev` version so it wont get reverted by accident in the future.

Reply:
@k4leung4 Done. Thank you!